src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -54,7 +55,20 @@ const AnnotationActivity = (props: Props) => {
     };
 
     const handleOnSelect = () => {
-        onSelect(id);
+        // Return an object of properties matching only the Annotation type
+        onSelect(
```

Comment:
If this is needed, can we use whitelist instead of a blacklist? The annotation shape is probably less likely to change than the component props. Or should we just pass in `item` rather than spreading it?